sklearn/linear_model/_base.py

```diff
@@ -419,6 +420,12 @@ class LinearRegression(MultiOutputMixin, RegressorMixin, LinearModel):
         ``-1`` means using all processors. See :term:`Glossary <n_jobs>`
         for more details.
 
+    solver : str, default="svd"
+        The solver to use. ``"svd"`` uses a Singular Value Decomposition-based
+        approach, by calling ``scipy.linalg.lstsq``. ``"cholesky"`` uses the
```

Comment:
This is confusing to me, I would assume ``svd`` to call ``linalg.svd`` but instead it uses scipy.lstsq which calls lapack, by default 'gelsd' to  solve LLS using divide-and-conquer SVD. This option is called ``'lsqr'`` in ``Ridge``, right? I think we should stick with the same meaning of names.

Reply:
Great point, I will keep things consistent and change `svd` to `lsqr`.